cmd/maya-exporter/app/command/command.go

```diff
@@ -1,18 +1,38 @@
 package command
 
 import (
+	"errors"
 	goflag "flag"
+	"log"
+	"net/url"
 
 	"github.com/golang/glog"
+	"github.com/openebs/maya/cmd/maya-exporter/app/collector"
 	"github.com/openebs/maya/pkg/util"
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/spf13/cobra"
 )
 
+// Constants defined here are the dafault value of the flags. Which can be
+// changed while running the binary.
+const (
+	// listenAddress is the address where exporter listens for the rest api
+	// calls.
+	listenAddress = ":9500"
+	// metricsPath is the endpoint of exporter.
+	metricsPath = "/metrics"
+	// controllerAddress is the address where jiva controller listens.
+	controllerAddress = "http://localhost:9501"
+	// storageEngine the component which provides the storage on cloud.
+	storageEngine = "jiva"
```

Comment:
Should we have this as storageEngineType? or call it "casType" and the comment can say: "The type of container attached storage (CAS) from which the metrics need to be exported. Default is Jiva"

Reply:
If the exporter is not meant to get metrics from local PV, "casType" seems apt.